schema.py

```diff
@@ -233,10 +233,11 @@ class Schema(object):
     schema for the data that will be validated.
     """
 
-    def __init__(self, schema, error=None, ignore_extra_keys=False):
+    def __init__(self, schema, error=None, ignore_extra_keys=False, name=''):
```

Comment:
`name` should probably default to `None` since it makes it more obvious that the parameter was not set.

```suggestion
    def __init__(self, schema, error=None, ignore_extra_keys=False, name=None):
```

Reply:
That's how I actually originally had it implemented for the very reason you suggested, but ultimately figured not needing the additional conversion from `None` to `' '` would be beneficial. But having the added explicitly of converting it certainly wouldn't hurt and is simple enough to implement.

Reply:
I agree with @rmorshea here, we don't need this right now but it's wrong semantically and in the future we might need a distinction between the empty string and `None`, so I wouldn't want to make the API wrong here now.

Reply:
Default to `None` has been implemented